tests/test_cdb.py

```diff
@@ -90,5 +90,44 @@ def test_cui2snames_population(self):
             with self.subTest(cui):
                 self.assertIn(cui, self.undertest.cui2snames)
 
+
+    def test_merge_cdb(self):
+        # generating cdbs - two maker are requested as they point to the same created CDB. 
+        config = Config()
+        config.general["spacy_model"] = "en_core_web_md"
+        maker1 = CDBMaker(config)
+        maker2 = CDBMaker(config) # second maker is required as it will otherwise point to same object
+        path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "model_creator", "umls_sample.csv")
+        cdb1 = maker1.prepare_csvs(csv_paths=[path])
+        cdb2 = maker2.prepare_csvs(csv_paths=[path])
```

Comment:
That is how I assumed it would work.

Making two cdbs from the same maker seemed to point the same cdb1. i.e.

```
maker = CDBMaker(config)
path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "model_creator", "umls_sample.csv")
cdb1 = maker.prepare_csvs(csv_paths=[path])
cdb2 = maker.prepare_csvs(csv_paths=[path])
print(cdb1 == cdb2) 
```

returns true. 

I'm happy to change this if I'm mistaken or have overlooked something in the building. The same config seems to be fine for CDBMaker.

Reply:
I think you're right - it just modifies the same instance each time the `prepare_csvs` method is called.
Though merging two identical CDBs to return a third identical one could also be an edge case to test.

But what this means is that the way the entire class works is prone to side effects since in each `setUp` it looks like a _new_ `CDB` is generated, whereas in reality the same instance is modified and returned.